Examples/type-overrides-example/Package.swift

```diff
@@ -0,0 +1,39 @@
+// swift-tools-version:5.10
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftOpenAPIGenerator open source project
+//
+// Copyright (c) 2024 Apple Inc. and the SwiftOpenAPIGenerator project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftOpenAPIGenerator project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+import PackageDescription
+
+let package = Package(
+    name: "type-overrides-example",
+    platforms: [.macOS(.v14)],
```

Comment:
```suggestion
    platforms: [.macOS(.v10_15)],
```

I don't think we're using any new features in this example beyond the minimum platform version?